integrationTests/consensus/consensus_test.go

```diff
@@ -196,6 +198,8 @@ func TestConsensusBLSFullTest(t *testing.T) {
 		t.Skip("this is not a short test")
 	}
 
+	_ = logger.SetLogLevel("*:TRACE")
```

Comment:
if it is working. Delete this line and import of logger as well.

Reply:
removed